internal/Panels/Arrangers.js

```diff
@@ -129,6 +129,9 @@ const deferArrange = (config, keyframes, options) => {
 /**
  * Arranger that slides panels in from the right and out to the left.
  *
+ * This arranger is visually same as {@link ui/ViewManager.SlideLeftArranger} when transition but uses transform percentages instead of pixel values.
```

Comment:
It looks like `ui/ViewManager.SlideLeftArranger` already use percentages. So I think this comment should be revised.
If the visual is the same, what makes difference between `SlideLeftArranger` and `BasicArranger`?
I see BasicArranger uses `deferArrange` and `offset`. Please find more details about this.

Reply:
1. `offset` values here don't affect animation (https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats)
2. `deferArrange` was defined in https://github.com/enactjs/sandstone/pull/577 and it prevents skipping transition animations under system load. I updated the comments.